README.md

```diff
@@ -68,14 +79,18 @@ __Description:__
 
 The Kion CLI allows users to perform common Kion workflows via the command
 line. Users can quickly generate short term access keys (stak) via configured
-favorites or by walking through an account and role selection wizard.
+favorites or by walking through an account and role selection wizard. Users can
+similarly federate into the AWS console.
 
 __Commands:__
 
 ```text
 stak, s            Generate short-term access keys.
 
-favorite, fav, f   Access pre-configured favorites to quickly generate staks.
+console, con, c    Federate into the AWS console.
```

Comment:
This also is from an old version of the README.  We removed console access (for now) because it wasn't working.